packages/suite/src/components/suite/layouts/SuiteLayout/PageHeader/PageHeader.tsx

```diff
@@ -36,6 +36,12 @@ export const PageHeader = ({ backRoute, children }: PageHeaderProps) => {
     const selectedAccount = useSelector(selectSelectedAccount);
     // TODO subpages + tabs could be in some router config? this approach feels a bit fragile
     const isAccountTabPage = useSelector(selectIsAccountTabPage);
+    const routeName = useSelector(selectRouteName);
+
+    // handle moment when children are not rendered yet in the Trade section
```

Comment:
Why they are not yet rendered in trade section?

Reply:
A component is set by a `useEffect` in the `useLayout` and on the first render **TopMenu** is `undefined`. My guess.